nicsdru_origins_theme.theme

```diff
@@ -79,6 +79,7 @@ function nicsdru_origins_theme_preprocess_page(array &$variables) {
     'class' => ['layout-sidebar-first'],
     'id' => ['first_sidebar']
   ];
+  $variables['content_content_attr'] = [];
```

Comment:
`content_content`? Is there a more descriptive reference we can have here? Main content/central content/wrapper... naming stuff is hard :)

Reply:
refactored to 'content_wrapper_attr'